src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/AuditUtility.cs

```diff
@@ -52,12 +54,14 @@ internal class AuditUtility
         internal int TotalWarningsSuppressedCount { get; private set; }
 
         public AuditUtility(
-            ProjectModel.RestoreAuditProperties? restoreAuditProperties,
+            IEnumerable<DownloadDependency> downloadDependencies,
```

Comment:
Similar feedback, new args at the end. It's often ok if it things like cancellation token or loggers are last, so you can add it before that, or maybe even after graphs.